.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea/
```

Comment:
Can you remove this line?

Text editor files should be in local ignore.

Reply:
Thank you for your feedback. It has been removed now.